dotcom-rendering/src/server/htmlPageTemplate.ts

```diff
@@ -199,12 +201,13 @@ https://workforus.theguardian.com/careers/product-engineering/
 
                 <title>${title}</title>
                 <meta name="description" content="${he.encode(description)}" />
+				<meta charset="utf-8">
 				${
 					canonicalUrl !== undefined
 						? `<link rel="canonical" href="${canonicalUrl}" />`
 						: '<!-- no canonical URL -->'
 				}
-                <meta charset="utf-8">
+				${alternateLangLinks?.join('\n') ?? ''}
```

Comment:
```suggestion
				${alternateLangLinks?.join('\n') ?? '<!-- no alternate langs for the current page -->'}
```

Reply:
I think I will leave as is, it seems redundant for the normative case.